src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/PrefsFramework.md

```diff
@@ -0,0 +1,779 @@
+property of the [`model`](../FrameworkConcepts.md#model-objects) property” of the Panel.
+<aside class="infusion-docs-callout">
+[Models](../FrameworkConcepts.md#model-objects) are central to Infusion, which,
+while not formally a [Model-View-Controller framework](../FrameworkConcepts.md#mvc),
+embodies the the separation of concerns that is central to MVC.
+Most Infusion components have an internal model, for maintaining the state of the component.
+</aside>
+</li>
+<li> The value, `“default”`, is an EL path referencing the `“default”` property in the Primary Schema.</li>
+</ul>
+This  Preference Map is saying two things:
+<ol>
+<li>The preference called `“minEditor.autoPilot”` should be stored in the Panel’s model
+in a property called `autoPilot`, and</li>
+<li>the initial value for the property should be taken from the `“default”` property
+of the Primary Schema.</li>
+</ul>
+A Preference Map can specify other locations for Primary Schema information, besides the model.
+We'll see an example of this when we add antoher panel, later in this tutorial.
+</dd>
+<dt>`selectors`</dt>
+<dd>A Panel is a _[view component](../tutorial-gettingStartedWithInfusion/ViewComponents.md)_
+– a type of Infusion component that has a _view_, that is,
+a user interface. In order to maintain a separation between the code and the HTML for the view,
+the code interacts with the HTML through named [selectors](../ComponentConfigurationOptions.md#-selectors-): The code only references the name, and
+a Framework feature called the DOM Binder looks up the relevant DOM node for the name based
+on the information in this `selectors` option.
+
+Let’s look at this more closely:
+<pre class="highlight">
+<code class="hljs javascript">selectors: {
+    autoPilot: ".mec-autoPilot"
+},</code></pre>
+
+The content of a `selectors` property is a set of key/value pairs. The key is the ‘name’ of the
+selector and the value is the selector itself. This property has only one selector,
+named `autoPilot`. The value is the CSS selector `".mec-autoPilot"`.
+This selector references the actual checkbox in the template for the Panel.
+This template is found in the `html/autoPilot.html` file, which looks like this:
+
+<pre class="highlight">
+<code class="hljs html">&lt;section class="me-panel"&gt;
+    &lt;h2&gt;Auto-Pilot&lt;/h2&gt;
+
+    &lt;label for="minEditor-autoPilot"&gt;Enable the auto-pilot when the car starts&lt;/label&gt;
+    &lt;input type="checkbox" id="minEditor-autoPilot" class="mec-autoPilot"/&gt;
+&lt;/section&gt;</code></pre>
+
+You can see the `“mec-autoPilot”` class name on the `<input>` element.
+</dd>
+<dt>`protoTree`</dt>
+<dd>
+A Panel is also a _[Renderer component](../RendererComponents.md)_ – a type of Infusion component that uses the
+Infusion [Renderer](../Renderer.md) to render the view based on data in the component’s model.
+The _[protoTree](../RendererComponentTrees.md)_ is the instructions for how the data in the component’s model maps to the template.
```

Comment:
For the prototree link can we set the url to "../RendererComponentTrees.md#prototrees"?